code/datums/elements/bane.dm

```diff
@@ -33,7 +33,15 @@
 	qdel(target.GetComponent(/datum/component/on_hit_effect))
 	return ..()
 
-/datum/element/bane/proc/check_bane(mob/living/bane_applier, atom/target)
+/datum/element/bane/proc/check_bane(bane_applier, target, bane_weapon)
+	if(!check_biotype_path(bane_applier, target))
+		return
+	var/atom/movable/atom_owner = bane_weapon
+	if(SEND_SIGNAL(atom_owner, COMSIG_OBJECT_PRE_BANING, target) & COMPONENT_CANCEL_BANING)
+		return
+	return TRUE
+
+/datum/element/bane/proc/check_biotype_path(mob/living/bane_applier, atom/target)
 	if(!isliving(target))
```

Comment:
This proc should be changed to have a proper boolean dichotomy (`TRUE`/`FALSE`) in its whole rather than the silly `TRUE`/`FALSE`/`null` thing it now has. Also DMdoc please, just for stat purposes